pkg/cmd/template/schema_author_test.go

```diff
@@ -521,12 +521,12 @@ Invalid schema - @schema/default is wrong type
 
 schema.yml:
     |
+  3 | #@schema/default [{"item": 1}]
   4 | map: thing
     |
 
-    = found: array
+    = found: array (by schema.yml:3)
     = expected: string (by schema.yml:4)
-    = hint: is the default value set using @schema/default?
```

Comment:
Thank you, hints, you did your service. Also, delighted to see us evolving passed this form. 👍🏻